### ISSUE TITLE
🐞 BUG - Icon list widget when using (em) for text size plus link - changes text size 

### ISSUE THREAD

**Issue by dannygrio** [elementor/elementor#14829]
- [ x ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

When I work with icon list and my font size is set to (em) the Icon list and text behaves correctly. However, as soon as I try to add a link to the icon list item, the text blows up in size.  The link size under, site settings, typography is set to body text (1em) on normal and hover, just to make sure.  Fresh clean wordpress with only elementor installed and replicated this issue in two different servers and websites, with different themes and also native theme. 

## Steps to reproduce

- Create a section
- Drop an Icon List widget 
- Under Style / Text - give a typography size of 3,5 em 
- Under content / icon list / Type inside item link area anything

This should cause the font to blow up in size. 

## Isolating the problem

- [ x ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ x ] This bug happens with a default WordPress theme active.
- [x ] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.4.15
	PHP Version: 7.2.34
	PHP Max Input Vars: 5000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.7.1
	Site URL: https://teste.dannyg.com.br
	Home URL: https://teste.dannyg.com.br
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 256M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Twenty Twenty-One
	Version: 1.3
	Author: the WordPress team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.93 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.2.3
		Author: Elementor.com

	WP Reset
		Version: 1.86
		Author: WebFactory Ltd


== Elementor - Compatibility Tag ==
	


== Elements Usage ==
	
	wp-page : 1
		icon-list : 1


== Log ==
	
Log: showing 1 of 12021-05-05 12:54:03 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.2',
  'to' => '3.2.3',
)]

				
</details>

**Comment by DorShahar** [comment]
Hi @dannygrio 
Thanks for reporting this issue, we will take care of it.

Thanks.

**Comment by shilo-ey** [comment]
This tweak has been implemented in Elementor v3.4.0

Feel free to update

Thanks!
